benches/compress.rs

```diff
@@ -82,6 +82,14 @@ fn bench_dbtext(c: &mut Criterion) {
             b.iter(|| unsafe { compressor.compress_into(&buf, &mut buffer) });
         });
 
+        unsafe { compressor.compress_into(&buf , &mut buffer); };
```

Comment:
Super nit: you have an extra space before the comma.

You can just run 

cargo fmt

And that should clean it up